packages/showcases/web-components/package.json

```diff
@@ -20,7 +20,7 @@
     "@storybook/addon-links": "^6.1.11",
     "@storybook/web-components": "^6.1.11",
     "@vtmn/web-components": "*",
-    "babel-loader": "^8.2.2",
+    "babel-loader": "8.1.0",
```

Comment:
You have to downgrade for a compliance that's it ?

Reply:
Yes and alignment through babel loaders in other packages...